src/nav/kubernetes-pixie.yml

```diff
@@ -3,14 +3,22 @@ path: /docs/kubernetes-pixie
 pages:
   - title: Introduction to Kubernetes integration
     path: /docs/kubernetes-pixie/kubernetes-integration/get-started/introduction-kubernetes-integration
-  - title: Install Kubernetes integration
-    path: /docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure
-  - title: Manual install using Helm
-    path: /docs/kubernetes-pixie/kubernetes-integration/installation/install-kubernetes-integration-using-helm
-  - title: Install Kubernetes in Windows
-    path: /docs/kubernetes-pixie/kubernetes-integration/installation/install-kubernetes-integration-windows
+  - title: New Relic Kubernetes components
+    path: /docs/kubernetes-pixie/kubernetes-integration/get-started/kubernetes-components
   - title: Compatibility and requirements
     path: /docs/kubernetes-pixie/kubernetes-integration/get-started/kubernetes-integration-compatibility-requirements
+  - title: Install Kubernetes integration
+    pages:
+      - title: Get started with Kubernetes
+        path: /docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure
+      - title: Manual install using Helm
+        path: /docs/kubernetes-pixie/kubernetes-integration/installation/install-kubernetes-integration-using-helm
+      - title: Install EKS Fargate
+        path: /docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/install-fargate-integration
+      - title: Install Kubernetes in Windows
+        path: /docs/kubernetes-pixie/kubernetes-integration/installation/install-kubernetes-integration-windows
+      - title: Upgrading Kubernetes integration
```

Comment:
@nbaenam I would add "the" like this: "Upgrading the Kubernetes integration."

Reply:
Done!